msg/InertialParameters.msg

```diff
@@ -0,0 +1,12 @@
+# This message describes the inertial parameters of a single rigid body.
+#
+# The inertial parameters of a single rigid body defined by:
+#  * the name of the rigid body
+#  * the inertial parameters vector [m, h_x, h_y, h_z, I_{xx}, I_{xy}, I_{yy}, I_{xz}, I_{yz}, I_{zz}]^T,
```

Comment:
This documentation is not correct as `geometry_msgs/Inertia` receives the barycenter position
```suggestion
#  * the inertial parameters vector [m, c_x, c_y, c_z, I_{xx}, I_{xy}, I_{yy}, I_{xz}, I_{yz}, I_{zz}]^T,
```